API_Guidelines.md

```diff
@@ -81,13 +89,20 @@ Payload
 :
 ```javascript
 {
-  "movies": {
+  "movie": {
     "id": 2,
     "title": "Goodfellas",
     "year": "1990"
   }
 }
 ```
+Payload (If no data is found, then `findRecord` returns an null value)
+:
+```javascript
+{
+  "movie": null
```

Comment:
I thought that a 404 should be returned also, Anthony. However, the RESTAdapter documentation says that any response other than 2xx or 304 is considered a "failure". https://emberjs.com/api/data/classes/DS.RESTAdapter.html#toc_success-and-failure

And the `handleResponse` adapter method documentation suggests that an API could return a 200 status with an error message. https://emberjs.com/api/data/classes/DS.RESTAdapter.html#method_handleResponse